### ISSUE TITLE
SNP rsID

### ISSUE THREAD

**Issue by rlesurf** [genome/civic-client#353]
For things like germline polymorphisms, it would be good to have a field to put the rsID.

**Comment by obigriffith** [comment]
Agreed. For now we have been including it in the text of the evidence statement or as the actual variant name in at least one case.

**Comment by kkrysiak** [comment]
Maybe it could be displayed as a parenthetical after the variant origin? Otherwise we will be adding another display field that will >90% of the time be NA. 

Also, should we allow a comma-separated list?

**Comment by kkrysiak** [comment]
I suppose we could see about linking out to dbSNP. Pulling in MAF information would be awesome but add another level of complication. If we did have another field I'd propose it goes after the Drug field. At least then we'd have 5 fields on each side.

**Comment by jmcmichael** [comment]
It's not a problem for me to add this field to the forms and views, @acoffman just needs to add it on the server.

**Comment by malachig** [comment]
I wonder if we should create a special field for RSID or simply implement the variant synonym concept and use that to store this and many other things.  RSID is one of many IDs that we might want to associate with a variant...

**Comment by malachig** [comment]
Also we would get this info for free by interfacing with MyVariantInfo, along with a lot of other useful info...

**Comment by malachig** [comment]
I believe following issue is a more generic solution to this problem that will allow us to benefit from the efforts of the MyVariantInfo team and get RSID along with a ton of other great stuff. 

https://github.com/genome/civic-server/issues/171

Unless there are objections we can probably close this one.
